definitions/npm/react-with-separator_v1.x.x/flow_v0.72.x-/react-with-separator_v1.x.x.js

```diff
@@ -0,0 +1,26 @@
+type WithSeparatorProps = {
+  children?: React$Node,
+  leading?: boolean,
+  separator: string | React$Element<any>,
```

Comment:
Can you explain why you went with `React$Element<any>` instead of `React$Node`?

Reply:
`cloneElement` only accepts `React$Element` type - https://github.com/facebook/flow/blob/41b0eab99cdc5199421f7cccad9e0c4950f8b2f9/tests/react_custom_funs/clone_element.js

The library explicitly handles the string case - https://github.com/LinusU/react-with-separator/blob/0027cc509653a825cfc77e8963e17fef05acb50f/index.js#L6